core/src/apps/bitcoin/get_public_key.py

```diff
@@ -79,9 +79,26 @@ async def get_public_key(
     )
 
     if msg.show_display:
-        from trezor.ui.layouts import show_xpub
+        from trezor.ui.layouts import show_pubkey
+        from .keychain import address_n_to_name
+        from apps.common.paths import address_n_to_str
 
-        await show_xpub(node_xpub, "XPUB")
+        path = address_n_to_str(address_n)
+        account_name = address_n_to_name(coin, address_n, script_type)
+        if account_name is None:
+            account = "Unknown path"
```

Comment:
`Unknown path` or `Unknown account`?

Reply:
`Unknown account` is more user friendy.

Reply:
9fda87ed8bbd0d362d51c4b45e1d18edfd41419b